Index: src/main/java/com/opencart/driver/DriverRepository.java
===================================================================
--- src/main/java/com/opencart/driver/DriverRepository.java	(revision a38eb5112502fc14b14e126ff49685d9c97a69eb)
+++ src/main/java/com/opencart/driver/DriverRepository.java	(revision a38eb5112502fc14b14e126ff49685d9c97a69eb)
@@ -1,28 +0,0 @@
-package com.opencart.driver;
-
-import io.github.bonigarcia.wdm.WebDriverManager;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.chrome.ChromeDriver;
-
-public class DriverRepository {
-
-    public static final ThreadLocal <WebDriver> DRIVERS = new ThreadLocal<>();
-    public static WebDriver webDriver;
-
-    private DriverRepository () {
-    }
-
-    public static void downloadWebDriver () {
-        WebDriverManager.chromedriver().setup();
-    }
-
-    public static void instanceWebBrowser () {
-        webDriver = new ChromeDriver();
-        DRIVERS.set(webDriver);
-    }
-
-    public static void closeBrowser () {
-        webDriver.quit();
-    }
-
-}
Index: src/main/java/com/opencart/navigation/Navigation.java
===================================================================
--- src/main/java/com/opencart/navigation/Navigation.java	(revision a38eb5112502fc14b14e126ff49685d9c97a69eb)
+++ src/main/java/com/opencart/navigation/Navigation.java	(revision a38eb5112502fc14b14e126ff49685d9c97a69eb)
@@ -1,19 +0,0 @@
-package com.opencart.navigation;
-
-import com.opencart.driver.DriverRepository;
-import org.openqa.selenium.WebDriver;
-
-public class Navigation {
-    private WebDriver driver;
-
-    public Navigation () {
-        driver = DriverRepository.DRIVERS.get();
-    }
-
-    public void navigateToURrl (String url) {
-        driver.get(url);
-        driver.manage().window().maximize();
-    }
-}
-
-
Index: src/test/java/BaseTest.java
===================================================================
--- src/test/java/BaseTest.java	(revision a38eb5112502fc14b14e126ff49685d9c97a69eb)
+++ src/test/java/BaseTest.java	(revision a38eb5112502fc14b14e126ff49685d9c97a69eb)
@@ -1,23 +0,0 @@
-import com.opencart.driver.DriverRepository;
-import org.testng.annotations.AfterSuite;
-import org.testng.annotations.BeforeClass;
-import org.testng.annotations.BeforeSuite;
-
-public class BaseTest {
-
-    @BeforeSuite
-    public void setup () {
-        DriverRepository.downloadWebDriver();
-    }
-
-    @BeforeClass
-    public void createDriver () {
-        DriverRepository.instanceWebBrowser();
-    }
-
-    @AfterSuite
-    public void closeBrowser () {
-        DriverRepository.closeBrowser ();
-    }
-
-}
